components/media_player/ui/CardTVChannelList

```diff
@@ -0,0 +1,253 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2018-2019 Marton Borzak <hello@martonborzak.com>
```

Comment:
Please change the copyright to you. I didn't write this code :)

Reply:
resolved